### PR TITLE
Updates to change/governance process

### PR DESCRIPTION
**Pull request by josee-sabourin**
In response to [issue 163](https://github.com/NABSA/gbfs/issues/163), we are hoping to move forward with updating our change process. This [document](https://docs.google.com/document/d/1LIQECzeKw7nVPiUVJpCzkmSkmWcmwX5FePzA8y4QQcM/edit?usp=sharing) outlines the current change process as per the repo, the proposed change process in issue 163, the de facto change process we have been more or less following, as well as what MobilityData would recommend.

One of our open questions is whether or not to include a deadline for implementing an RC, or how long something should remain an RC. There are instances of changes taking a considerable amount of time to be implemented. We understand that resources may be reduced due to COVID-19 which may cause delay, which is why we wanted to discuss whether or not a deadline would be beneficial or if it would create added stress and complications. Further, what would be the process if that deadline wasn't met?

Please let us know your thoughts on this proposed process as well as on the question above!

### CODE REVIEW

**Comment by rms-consult** [comment]
Hello!

I went quickly over the proposed modification of the change process. Sounds to me surprisingly rigid. The time frame for votes e.g. of just one week is rather tight.

The implementation requirement for having changes moved from RC to release kept me wondering. So the proposed changes first need to be implemented by one producer and one consumer in order to be moved to the official release. Tracking the implementation on the producer side seems straight forward. Just check the feed they produce. But on the consumer side it doesn't make sense to me. How do you track if the changes were implemented. A consumer doesn't produce a feed and not necessarily is the software available on github. Sounds to me more like a rhetorical question.

And what about votes against a proposed change? Do they count at all?

Maybe the proposed modification of the change process becomes clearer to me if you could elaborate a little bit better what the relevant lessons from GTFS were/are. I didn't follow GTFS closely enough so it would be helpful to understand what the issues were and when & how they were fixed. Thx.

**Comment by matt-wirtz** [comment]
Hello!

I went quickly over the proposed modification of the change process. Sounds to me surprisingly rigid. The time frame for votes e.g. of just one week is rather tight.

The implementation requirement for having changes moved from RC to release kept me wondering. So the proposed changes first need to be implemented by one producer and one consumer in order to be moved to the official release. Tracking the implementation on the producer side seems straight forward. Just check the feed they produce. But on the consumer side it doesn't make sense to me. How do you track if the changes were implemented. A consumer doesn't produce a feed and not necessarily is the software available on github. Sounds to me more like a rhetorical question.

And what about votes against a proposed change? Do they count at all?

Maybe the proposed modification of the change process becomes clearer to me if you could elaborate a little bit better what the relevant lessons from GTFS were/are. I didn't follow GTFS closely enough so it would be helpful to understand what the issues were and when & how they were fixed. Thx.

**Comment by mplsmitch** [comment]
Is a simple majority enough to move forward with a proposal?  Suppose a proposed change receives 10 votes in favor and 9 against, does it pass? What happens in the case of a tie?

The proposal makes clear that "Anyone can call for a vote" but doesn't define who can make a proposal (anyone?) or if there are any voting requirements. Can anyone vote or do they need to be affiliated with a producer or consumer?  Can multiple representatives of an organization vote on a proposal or would organizations be limited to a single vote?

**Comment by josee-sabourin** [comment]
Hi all! 
We've updated the proposal to answer questions and comments from the community, the proposal document linked above and [here](https://docs.google.com/document/d/1LIQECzeKw7nVPiUVJpCzkmSkmWcmwX5FePzA8y4QQcM/edit?usp=sharing) has also been updated with more information about the origins of GBFS governance as it relates to GTFS. If you are interested in learning more about GTFS Governance, an explainer document on its history is coming soon. 

Let us know if you have any questions or concerns! 

@matt-wirtz @mplsmitch

**Comment by josee-sabourin** [comment]
I hereby call a vote on this proposal. Voting will be open for 7 full calendar days until 11:59PM UTC on Monday, November 9th, 2020.
Please vote for or against the proposal, and include the organization for which  you are voting in your comment.

File: README.md [hunk 1]
```diff
@@ -48,13 +48,23 @@ The data in the specification contained in this document is intended for consump
...
-3.	Find at least one GBFS producer to implement and test the proposed change.
-4.	Submit a final request-for-comments on the proposed change to the issue discussion. If no outstanding issues are identified after one week’s time, and there is general agreement that the proposed change is worthwhile and follows the GBFS guiding principles outlined below, the proposal will be officially adopted.
+The specification is not fixed or unchangeable. As the shared mobility industry evolves, it is expected that the specification will be extended by the GBFS community to include new features and capabilities over time. To manage the change process, the following guidelines have been established.
+
+
+* A change is proposed by opening a Pull Request at the GBFS GitHub repository.A change is proposed by opening a Pull Request at the GBFS GitHub repository. The proposer becomes “The Advocate”. Comments and feedback from the GBFS community are received to iterate on the proposed change. Discussion lasts for as long as necessary to address questions and revisions, but must be at least 7 calendar days.
```
Comment (barbeau) [inline README.md]: "A change is proposed by opening a Pull Request at the GBFS GitHub repository" is repeated here

File: README.md [hunk 2]
```diff
@@ -48,13 +48,23 @@ The data in the specification contained in this document is intended for consump
...
+  * *I hereby call a vote on this proposal. Voting will be open for 10 full calendar days until 11:59PM UTC on X.<br /> Please vote for or against the proposal, and include the organization for which you are voting in your comment. <br /> Please note if you can commit to implementing the proposal.*
+* The person calling for the vote should announce the vote in the [GBFS Slack channel](https://mobilittydata-io.slack.com) with a link to the PR. The message should conform to this template:
+  * *A vote has been called on PR # [title of PR] (link to PR). This vote will be open for 10 full calendar days, until 11:59PM UTC on X. Please vote for or against the proposal on GitHub.*
+* MobilityData will both comment on the PR on GitHub and send a reminder in the GBFS Slack channel when there are 2 calendar days remaining on the vote. The reminder should conform to this template: 
+  * Slack: <br />*Voting on PR # [title of PR] (link to PR) closes in 2 calendar days. Please cote for or against the proposal on GitHub.*
```
Comment (barbeau) [inline README.md]:
```suggestion
  * Slack: <br />*Voting on PR # [title of PR] (link to PR) closes in 2 calendar days. Please vote for or against the proposal on GitHub.*

File: README.md [hunk 3]
```diff
@@ -48,13 +48,23 @@ The data in the specification contained in this document is intended for consump
...
+* The person calling for the vote should announce the vote in the [GBFS Slack channel](https://mobilittydata-io.slack.com) with a link to the PR. The message should conform to this template:
+  * *A vote has been called on PR # [title of PR] (link to PR). This vote will be open for 10 full calendar days, until 11:59PM UTC on X. Please vote for or against the proposal on GitHub.*
+* MobilityData will both comment on the PR on GitHub and send a reminder in the GBFS Slack channel when there are 2 calendar days remaining on the vote. The reminder should conform to this template: 
+  * Slack: <br />*Voting on PR # [title of PR] (link to PR) closes in 2 calendar days. Please cote for or against the proposal on GitHub.* 
+  * GitHub:<br />*Voting on this PR closes in 2 calendar days. Please vote for or against hte proposal, and include the organization for which you are voting in your comment. Please note if you can commit to implementing the proposal.*
```
Comment (barbeau) [inline README.md]:
```suggestion
  * GitHub:<br />*Voting on this PR closes in 2 calendar days. Please vote for or against the proposal, and include the organization for which you are voting in your comment. Please note if you can commit to implementing the proposal.*
```

File: README.md [hunk 4]
```diff
@@ -48,13 +48,23 @@ The data in the specification contained in this document is intended for consump
...
+* Once a vote is called, a "Vote Open" label will be added to the PR. After the 2 day reminder, the label will be replaced with "Vote Closing Soon", once the vote is closed, the label will become either “Vote Passed” or “Voted Failed” depending on the vote outcome.
+* At least 3 votes in favor, in addition to the author of the Pull Request, are required for a proposal to pass. At least one of these votes must be from a producer and at least one from a consumer.
+* Implementation requirements are that both 1 producer and 1 consumer commit to implementing the changes. A change is placed into RC status pending implementation. Once implemented successfully, the change is merged into an official release. .
```
Comment (barbeau) [inline README.md]:
```suggestion
* Implementation requirements are that both 1 producer and 1 consumer commit to implementing the changes. A change is placed into RC status pending implementation. Once implemented successfully, the change is merged into an official release.
```

File: README.md (hunk 1 above)
Comment (josee-sabourin) [reply README.md]: Fixed!

File: README.md [hunk 5]
```diff
@@ -48,13 +48,25 @@ The data in the specification contained in this document is intended for consump
...
+* MobilityData will both comment on the PR on GitHub and send a reminder in the GBFS Slack channel when there are 2 calendar days remaining on the vote. The reminder should conform to this template: 
+  * Slack: <br />*Voting on PR # [title of PR] (link to PR) closes in 2 calendar days. Please vote for or against the proposal on GitHub.* 
+  * GitHub:<br />*Voting on this PR closes in 2 calendar days. Please vote for or against the proposal, and include the organization for which you are voting in your comment. Please note if you can commit to implementing the proposal.*
+* Once a vote is called, a "Vote Open" label will be added to the PR. After the 2 day reminder, the label will be replaced with "Vote Closing Soon", once the vote is closed, the label will become either “Vote Passed” or “Voted Failed” depending on the vote outcome.
+* At least 3 votes in favor, in addition to the author of the Pull Request, are required for a proposal to pass. At least one of these votes must be from a producer and at least one from a consumer.
+* The advocate should cancel a vote if significent changes are made to the proposal after stakeholders have already voted.
```
Comment (evansiroky) [inline README.md]: typo: `significent`

Comment (josee-sabourin) [reply README.md]: fixed thanks!

**Comment by cmonagle** [comment]
+1 on behalf of Transit.

**Comment by evansiroky** [comment]
+1 from IBI Group

**Comment by kanagy** [comment]
+1 from Google Maps

**Comment by josee-sabourin** [comment]
Given that this PR proposes extending voting time to 10 days, and the general support on the PR, we will use this PR as a "practice" on the extended voting period. Thus, voting will be open until **11:59PM UTC on Thursday, November 12th**.

**Comment by jayjonsson** [comment]
+1 from Lime

**Comment by bhandzo** [comment]
+1 from Bird

**Comment by sfaubert1** [comment]
+1 from PBSC

**Comment by josee-sabourin** [comment]
The vote on this is now closed, and it passes!

Votes in favor:
Transit (consumer)
IBI (consumer)
Google Maps (consumer)
Lime (producer)
Bird (producer)
PBSC (producer)

No votes against.

**Pull request merged**